### ISSUE TITLE
Initiative Tracker

### ISSUE THREAD

**Issue by erikblessman** [marcojanssen/drunagor-helper#30]
An Initiative Tracker would be amazing and much appreciated.

The biggest problem we have with the physical initiative tracker is mapping minis to their cards on the tracker.

Suggested Features:
- Allow markers to match the color of the base(s) for easy reference to the physical map
- Allow for condition markers
- Tie-in to the characters of an existing campaign
- Auto-Draw Rune Tile(s)

**Comment by marcojanssen** [comment]
I agree with you. Started it a while back but discontinued it considering Drunagor has a lot of variables for each scenario making it really hard to properly build. 

Have been working hard on the apocalypse content which has just been released. Might pick it up later on but for now I'm going to work on different projects.

**Comment by erikblessman** [comment]
Thanks for the response, but mostly for all of the work you're putting into
this!

**Comment by erikblessman** [comment]
FYSA - I did fork this repo to start an integrated initiative tracker and monster health/condition tracker. If you're interested don't hesitate to reach out.
